.azure-pipelines/templates/ccache.yml

```diff
@@ -0,0 +1,18 @@
+steps:
+  - bash: |
+      sudo apt-get install -y ccache
```

Comment:
We are going to get this to work for OSX too? I think we could get this from `conda`

Reply:
good idea. I suggest to leave it as is for this PR since it seems to be working, and every time you seem to touch `ccache` it breaks again. We can start using `conda` in a follow-up PR?